pkg/webui/components/list/index.js

```diff
@@ -115,6 +139,7 @@ List.defaultProps = {
   bordered: false,
   emptyMessage: sharedMessages.noMatch,
   emptyMessageValues: {},
+  header: null,
```

Comment:
The `renderItem` function you have in the story should be the default prop here.

Reply:
I meant this one:
```js
  renderItem (item, index) {
    return (
      <List.Item key={index}>
        {item}
      </List.Item>
    )
  }
```